code/modules/mob/living/carbon/xenomorph/abilities.dm

```diff
@@ -160,6 +160,9 @@
 
 	var/wait_time = 2 SECONDS - max(0, (1 SECONDS * X.health / X.maxHealth)) //Between 1 and 2 seconds, depending on health.
 
+	if(isxenohivemind(X)) // Hivemind has free build, it should be slower
```

Comment:
Could you turn the `wait_time` base into an ability var, and give the hivemind a child with a higher one?
Don't think the hivemind building speed according to health is as relevant as it is for a xeno.